internal/pkg/deploy/cloudformation/stack/worker_svc_test.go

```diff
@@ -67,7 +67,7 @@ Outputs:
     Value: hello`,
 				}
 			},
-			wantedErr: fmt.Errorf("convert the sidecar configuration for service frontend: %w", errors.New("cannot parse port mapping from 80/80/80")),
+			wantedErr: fmt.Errorf("exposed ports configuration for service frontend: cannot parse port mapping from 80/80/80"),
```

Comment:
I think we should keep the original test case, and have a new test case for the "exposed port" error

Reply:
So the thing here is that we send `exposedPorts` to the `convertSidecar` in `stack/lb-web-svc.go` . And in order `exposedPorts` we call `ExposedPorts()` method from the manifest which also parses the sidecar container ports in order to organize `[]exposedPort` for us. So in this test case before even `convertSidecar` fail for the same parsing reason, `ExposedPorts()` fails and responds with the new error. Does that make sense? So even if we create a new test case for the exposedports this test case will still give the changed error i.e. `exposed ports configuration ....`

Reply:
I meant we want to change the manifest input for the original test case so that the old error message is testable :D

Reply:
No, I am not saying that anything after `ExposedPorts()` is untestable. The only thing is we are making `manifest.ParsePortMapping(config.Port)` call in the `convertSidecar` as well as `ExposedPorts()` but because `ExposedPorts()` is called before `convertSidecar` now we are getting that error from `ExposedPorts()`. 
I don't think we can modify the manifest in a way that would error for `manifest.ParsePortMapping(config.Port)` call in the `convertSidecar` but not in the `ExposedPort()`

Reply:
Does it mean `cannot parse port mapping` is dead code?

Reply:
Nope, it is not a dead code as we can see `cannot parse port mapping` in `wantedErr` in the above code snippet.